participants/Novel-Wiki.md

```diff
@@ -0,0 +1,62 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+*Novel-Wiki,the goal is to create a wiki catalog of works of fiction.People can retrieve novels uploaded from anyone for free.We hope that we can improve this catalogue through your efforts,in the spirit of freedom and openness.*
```

Comment:
Please ensure that the books you upload are past copyright or are publicly licensed.